src/manifest.json

```diff
@@ -9,42 +9,406 @@
+      "args": {
+        "initialIsOpen": false
+      },
+      "children": [
+        {
+          "type": "internal-link",
+          "id": "sq-intro",
+          "label": "Introduction",
+          "summary": "Try real-time searching and perform complex structured queries without compromising the database performance.",
+          "args": {
+            "path": "/sq/introduction.md"
+          }
+        },
+        {
+          "type": "internal-link",
+          "id": "sq-exact-match",
+          "label": "Exact match (TAG queries)",
+          "args": {
+            "path": "/sq/exact-match.md"
+          }
+        },
+        {
+          "type": "internal-link",
+          "id": "sq-full-text",
+          "label": "Full-text search (fuzzy, wildcards, synonyms, language support)",
+          "args": {
+            "path": "/sq/full-text.md"
+          }
+        },
+        {
+          "type": "internal-link",
+          "id": "sq-range",
+          "label": "Range queries (numeric)",
+          "args": {
+            "path": "/sq/range.md"
+          }
+        },
+        {
+          "type": "internal-link",
+          "id": "sq-geospatial",
+          "label": "Geospatial queries (GEOPOINT and GEOSHAPE)",
+          "args": {
+            "path": "/sq/geospatial.md"
+          }
+        },
+        {
+          "type": "internal-link",
+          "id": "sq-combined",
+          "label": "Combined queries (text + tag + numeric + geo)",
+          "args": {
+            "path": "/sq/combined.md"
+          }
+        },
+        {
+          "type": "internal-link",
+          "id": "sq-vector",
+          "label": "Vector search (range and KNN)",
+          "args": {
+            "path": "/sq/vector.md"
+          }
+        },
+        {
+          "type": "internal-link",
+          "id": "sq-aggregations",
+          "label": "Analytic and transformative (aggregations, load, groupby, sortby, apply, and filter)",
+          "args": {
+            "path": "/sq/aggregations.md"
+          }
+        },
+        {
+          "type": "internal-link",
+          "id": "sq-semantic",
+          "label": "Semantic search (pre-filtering + vectors)",
+          "args": {
+            "path": "/sq/semantic.md"
+          }
+        },
+        {
+          "type": "internal-link",
+          "id": "sq-learn-more",
+          "label": "Learn more",
+          "args": {
+            "path": "/sq/learn-more.md"
+          }
+        }
+      ]
+    },
+    {
+      "type": "group",
+      "id": "tf",
+      "label": "Triggers and functions explained",
       "args": {
         "initialIsOpen": false
       },
       "children": [
         {
           "type": "internal-link",
-          "id": "working_with_json",
-          "label": "Working with JSON",
+          "id": "tf-intro",
+          "label": "Introduction",
+          "summary": "Trigger and execute JavaScript functions in the Redis process.",
```

Comment:
@dwdougherty , please check the summary for trigger and functions.

An example is attached below. The summary can be added only per file, so I added it only for the "introduction" file to display some context for users to try it, other files, imo, should be self-explaining.
I'll also ask Thomas to review it once he accepts the invitation.

<img width="410" alt="image" src="https://github.com/RedisInsight/Tutorials/assets/99594890/82b51c98-8874-434c-bf34-1f108f36d245">

Reply:
LGTM, but Thomas should give it a look.